doc/user/content/sql/recursive-ctes.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Recursive CTEs
+description: "Recursive CTEs is used to define recursive queries."
+menu:
+  main:
+    parent: 'transform'
+    name: Recursive CTEs
+    weight: 22
+---
+
+
+Recursive CTEs can be used to define computations on recursively defined strucutes (such as trees or graphs) implied by the data in your tables.
+
+## Syntax
+
+### with_recursive_cte
+
+{{< diagram "with-recursive-ctes.svg" >}}
+
+### recursive_cte_binding
+
+{{< diagram "recursive-cte-binding.svg" >}}
+
+Field | Use
+------|-----
+**RETURN AT RECURSION LIMIT $n** | An optional clause indicating that the fixpoint computation should stop after `$n` iterations and offer the current values computed for each `recurive_cte_binding` to the `select_stmt`. This could be useful when debugging and validating the correctness of recursive queries. See the [Examples](#examples)
```

Comment:
`.` missing at the end.

Also, maybe add to the end of the "useful when" sentence something like "or when you know exactly how many iterations you want to have, regardless of reaching a fixpoint".